### PR TITLE
added tests for start/stop/suspend/resume/restart service cmdlets

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Part of https://github.com/PowerShell/PowerShell/issues/4147

### CODE REVIEW

File: test/powershell/Modules/Microsoft.PowerShell.Management/ControlService.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+      if ($wasStopped) {
+        Stop-Service Spooler
+      }
+    }
+  }
+
+  It "Suspend/Resume-Service works" {
+    $workstation = Get-Service LanmanWorkstation
+    $workstation | Should Not BeNullOrEmpty
+    Suspend-Service LanmanWorkstation
```
Comment (TravisEz13) [inline test/powershell/Modules/Microsoft.PowerShell.Management/ControlService.Tests.ps1]: Can we use something else other than the workstation service?

**Review by TravisEz13 — CHANGES_REQUESTED** [review]
I'd really prefer that tests not change the state of the workstation service at all.

File: test/powershell/Modules/Microsoft.PowerShell.Management/ControlService.Tests.ps1 (hunk 1 above)
Comment (SteveL-MSFT) [reply test/powershell/Modules/Microsoft.PowerShell.Management/ControlService.Tests.ps1]: On my system, only 4 services support suspend/resume.  I'll use WER.

```powershell
PS C:\> get-service | ? { $_.CanPauseandcontinue -eq $true }

Status   Name               DisplayName
------   ----               -----------
Running  LanmanWorkstation  Workstation
Running  stisvc             Windows Image Acquisition (WIA)
Running  WerSvc             Windows Error Reporting Service
Running  Winmgmt            Windows Management Instrumentation
```

**Comment by SteveL-MSFT** [comment]
@TravisEz13 can you review again?

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**